src/Handlers/AbstractPaymentMethodHandler.php

```diff
@@ -695,4 +709,14 @@ private function getProduct(string $productId, Context $context): ProductEntity
 
         return $product;
     }
+
+    private function displaySafeErrorMessages(AdyenException $exception)
+    {
+        if (
+            $exception->getErrorType() == 'validation'
+            && in_array($exception->getAdyenErrorCode(), self::SAFE_ERROR_CODES)
+        ) {
+            $this->session->getFlashBag()->add('warning', $exception->getMessage());
```

Comment:
do you know maybe if the getMessage always human readable?

Reply:
I think the [error messages](https://docs.adyen.com/development-resources/error-codes) from the API are always human readable